migrations/test-fixtures/0012/03-symlink-already-exists/verify.sh

```
@@ -0,0 +1,25 @@
+
+# Pre-flight: passes (symlink is right target, version is 1.10.0)
+grep -q '^version: 1.10.0$' .claude/skills/agentic-apps-workflow/SKILL.md \
+  || { echo "PRE-FLIGHT 1 should pass (version 1.10.0)"; exit 1; }
+test -L "$HOME/.claude/skills/add-observability" \
+  || { echo "fixture 03 expects symlink to exist before verify"; exit 1; }
+readlink "$HOME/.claude/skills/add-observability" | grep -q '/agenticapps-workflow/add-observability$' \
+  || { echo "PRE-FLIGHT 4 — existing symlink should be correct target"; exit 1; }
+
+# Step 1 idempotent (returns 0): symlink already present, correct target
+test -L "$HOME/.claude/skills/add-observability" \
+  && readlink "$HOME/.claude/skills/add-observability" | grep -q '/agenticapps-workflow/add-observability$' \
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)